wireless/bluetooth/nimble/Make.defs

```diff
@@ -0,0 +1,39 @@
+############################################################################
```

Comment:
Wong header.  We should probably update the headers on the hello example, since I know I also use it for a base.

Reply:
done (also opened #457)